bedrock/privacy/templates/privacy/index.html

```diff
@@ -81,6 +81,7 @@ <h1 itemprop="name">{{ doc.h1.string }}</h1>
         <li class="policy-websites"><a href="{{ url('privacy.notices.websites') }}">{{ _('Mozilla Websites, Communications &amp; Cookies') }}</a></li>
         <li class="policy-firefox"><a href="{{ url('privacy.notices.firefox') }}">{{ _('Firefox Browser') }}</a></li>
         <li class="policy-firefox-os"><a href="{{ url('privacy.notices.firefox-os') }}">{{ _('Firefox OS') }}</a></li>
+        <li class="policy-firefox-cliqz"><a href="{{ url('privacy.notices.firefox-cliqz') }}">{{ _('Firefox + Cliqz') }}</a></li>
```

Comment:
@flodolo this adds a new string. The new notice page already has a german translation ready, so maybe we are ok to add this without a tag?